tests/acceptance/api_test.go

```diff
@@ -708,61 +722,6 @@ func TestApi(t *testing.T) {
 
 		})
 
-		t.Run("Should not be able to create lease if there's already an existing lease for the principal", func(t *testing.T) {
```

Comment:
I moved this test to an existing test flow, so we don't need to recreate the account and all that.